client/src/Interface.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import FilterButton from './FilterButton';
 
-const Interface = () => (
+const Interface = props => (
```

Comment:
I think our linter should have caught this?

https://github.com/airbnb/javascript#arrows--confusing